### PR TITLE
feat: hack db connection modal  ctabtns

### PR DESCRIPTION
**Pull request by pkdotson**
### SUMMARY
This pr creates two cta buttons to create dataset or to go to sqllab in the db connection modal after a user connects their db

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
after

https://user-images.githubusercontent.com/17326228/167940966-59f70052-dd76-4303-b9aa-724088ea21d3.mov

### TESTING INSTRUCTIONS
Go to database connection modal and connect a db. There should be two cta button that take you to create dataset or to sql lab.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by yousoph** [comment]
@jess-dillard can you see if the styling on the second step of the db connection screen looks good to you?

**Comment by jess-dillard** [comment]
@pkdotson @yousoph Let's make some styling changes. Make a quick Figma mock [here](https://www.figma.com/file/ZGRrjGFNGHYxlsqOQy7pz3/?node-id=1107%3A2558) (top part of modal only). Note there are some text tweaks as well.

Screen Shot 2022-05-20 at 1 30 22 PM

File: superset-frontend/src/components/DatabaseSelector/index.tsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ export type DatabaseObject = {
 type SchemaValue = { label: string; value: string };
 
 export interface DatabaseSelectorProps {
-  db?: DatabaseObject;
+  db?: DatabaseObject | null | undefined;
```
Comment (AAfghahi) [inline superset-frontend/src/components/DatabaseSelector/index.tsx:90]: I understand why it is null, but why does it also need to be undefined?

Comment (pkdotson) [reply superset-frontend/src/components/DatabaseSelector/index.tsx]: I think i was trying to appease ts in on the props. But I removed and no issue came about

**Review by AAfghahi — APPROVED** [review]

**Pull request merged**